src/screens/Exchange/ExchangeConfirm.js

```diff
@@ -243,12 +243,20 @@ class ExchangeConfirmScreen extends React.Component<Props, State> {
       to: recipient,
       contractAddress,
       data,
+      symbol: fromAssetSymbol,
     } = this.transactionPayload;
     const value = Number(amount || 0);
 
+    const isConvertedTx = isWethConvertedTx(fromAssetSymbol, contractAddress);
+
+    // for WETH converted txs on homestead, we need to provide ETH data or else estimation is always 0$
+    const contractAddressForEstimation = isProdEnv && isConvertedTx
+      ? '0x0000000000000000000000000000000000000000'
```

Comment:
Maybe import this from `ethers.constants.AddressZero` to make it less error prone?